CMSgov/onemac
# Oy2 16557

Story: https://qmacbis.atlassian.net/browse/OY2-16557
Endpoint: https://dei6jlsn7e65t.cloudfront.net/

### Details

Update text for "WIthdraw Package" action to "Withdraw"
Update text for "Package Withdrawn" status to "Withdrawn"

### Changes

- Updated action links
- Updated all places where status is shown
- Updated seed data

### Test Plan

1. Login as state submitter
2. Create or navigate to a SPA/Waiver
3. Verify in Package list the available action is "Withdraw"
4. Open the SPA/Waiver details page
5. Verify in the package actions list the action is named "Withdraw"
6. Click the Withdraw action
7. Verify the title in the popup is "Withdraw" and the action button is "Withdraw"
8. Click "Withdraw"
9. Find the SPA/Waiver in the package list and verify status is "Withdrawn"
10. Open the SPA/Waiver details page
11. Verify the Status in the Top Details Card is "Withdrawn"
